test/wadler_test.rb

```diff
@@ -610,4 +610,40 @@ def altshow(out)
       check_roundtrip(100, expected, builder_block)
     end
   end
+
+  describe 'must not break if no group was given' do
+    it 'must work with only a string' do
+      out = Oppen::Wadler.new
+      out.text 'Hello World'
+      _(out.output).must_equal 'Hello World'
+    end
+
+    it 'must work with only a breakable' do
+      out = Oppen::Wadler.new
+      out.breakable ''
+      _(out.output).must_equal ''
+    end
+
+    it 'must work with only a nest' do
+      out = Oppen::Wadler.new
+      out.nest(0) {
+        out.text 'Hello World'
+      }
+      _(out.output).must_equal 'Hello World'
+    end
+
+    it 'must work with multiple strings and breakables' do
+      out = Oppen::Wadler.new(width: 10)
+      out.text 'Hello World'
+      out.breakable
+      out.text 'How are you doig World'
+      out.breakable
+      out.text 'GoodBye World'
```

Comment:
```suggestion
      out
        .text('Hello World')
        .breakable
        .text('How are you doig World')
        .breakable
        .text('GoodBye World')
```

Use this style everywhere.

Reply:
This does not work. all Wadler methods do not return self.

Reply:
We should. In a different PR. And we need to change all tests to support it.

We also want `wrap/when` in here, and also `do`.

But let's not bother with it now. I will open a ticket.